modules/ROOT/examples/live-demos/paste-from-word/index.html

```diff
@@ -1,5 +1,5 @@
 <textarea id="paste-from-word">
-  <p><img style="float: right;" title="Tiny Logo" src="https://www.tiny.cloud/docs/images/logos/android-chrome-256x256.png" alt="TinyMCE Logo" width="128" height="128" /></p>
+  {{logofordemoshtmlright}}
   <h2>TinyMCE demo: PowerPaste plugin!</h2>
 <p>PREMIUM PLUGIN: The TinyMCE <a href="https://www.tiny.cloud/docs/enterprise/paste-from-word/">PowerPaste plugin</a> automatically cleans up and transfers content &amp; images from Microsoft Word and HTML sources to ensure clean, compliant content that matches the look and feel of the site.</p>
```

Comment:
closest content i could find
https://www.tiny.cloud/tinymce/6/powerpaste-support.html

Reply:
Again out of scope for this task. This should be handled by TINY-8302.